src/config.ml

```diff
@@ -2125,12 +2155,13 @@ let client_generate_connect_options t =
   let excerpt =
     Conf_map.filter
       (function
-        | B (Cipher, _) -> true
+        | B (Cipher, _) -> false (* TODO: only relevant if no NCP is used *)
         | B (Comp_lzo, _) -> true
         | B (Tun_mtu, _) -> true
         | B (Link_mtu, _) -> true
         | B (Pull, _) -> true
         | B (Tls_mode, `Client) -> true
+        | B (Auth, _) -> true (* TODO: only relevant if no NCP is used *)
```

Comment:
esp, with #36 in mind, I think the auth should be optional. but that can be addressed in a separate PR